### ISSUE TITLE
figure out if we can change model to only register mod key presses on key up

### ISSUE THREAD

**Issue by yanfali** [qmk/qmk_configurator#373]
The reason for doing this is to allow for shifted code input. So all the shifted codes on the number row can actually be input in the UI via the keyboard

**Comment by yanfali** [comment]
This actually seems to work. I will wait for numpad work to be completed
```javascript
// generate a keypress combo handler per keycode
function generateKeypressHandler(keycode) {
  const meta = store.getters['keycodes/lookupKeycode'](keycode.code);
  switch (meta.code) {
    case 'KC_LSFT':
    case 'KC_LGUI':
    case 'KC_LALT':
    case 'KC_LCTL':
      return {
        keys: keycode.keys,
        on_keyup: partial(keydownHandler, meta)
      };
    default:
      return {
        keys: keycode.keys,
        on_keydown: partial(keydownHandler, meta)
      };
  }
}
```

**Comment by yanfali** [comment]
Resolved by 472b910baaa9fb0fa5ab22d1c0380d66fec01e54
